modules/battery_manager/php/test.class.inc

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * This class implements a data Instance which represents a single
+ * test in the battery manager menu table.
+ *
+ * PHP Version 7
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Imaging
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\battery_manager;
+
+/**
+ * A Test represents a row in the Battery Manager menu table.
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Imaging
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+class Test implements \LORIS\Data\DataInstance
```

Comment:
Maybe TestInstance? I find the name "Test" confusing because it makes me think it's part of our test suite.